README.md

```diff
@@ -58,6 +58,8 @@ SSB-JS organization require discussion and consent.
 ### Contributors
 
 - 'Contributors' are anyone who participates in a project.
+- Contributors MUST read a project's CONTRIBUTING.md before submitting a pull request.
+- Contributors MUST update their pull requests until they have no blocking concerns.
```

Comment:
I have a small concern around this requirement -- more info (and a patch) here: https://github.com/ssb-js/ssb-js/pull/23

Reply:
Resolved